### ISSUE TITLE
Problem: Shibboleth URL definitions are broken

### ISSUE THREAD

**Issue by replaceafill** [archivematica/Issues#1667]
**Expected behaviour**

Archivematica and Storage Service work with Shibboleth authentication.

**Current behaviour**

Reported in https://github.com/artefactual-labs/ansible-archivematica-src/issues/358

This can be reproduced in the Compose development environment by adding `ARCHIVEMATICA_DASHBOARD_DASHBOARD_SHIBBOLETH_AUTHENTICATION: "true"` to environment variables of the `archivematica-dashboard` service, recreating the container and trying to log in:

```console
[2024-02-20 15:16:49 +0000] [87] [ERROR] Error handling request /
Traceback (most recent call last):
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/middleware/locale.py", line 24, in process_request
    ) = is_language_prefix_patterns_used(urlconf)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/conf/urls/i18n.py", line 31, in is_language_prefix_patterns_used
    for url_pattern in get_resolver(urlconf).url_patterns:
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/urls/resolvers.py", line 715, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/urls/resolvers.py", line 708, in urlconf_module
    return import_module(self.urlconf_name)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/src/src/dashboard/src/urls.py", line 49, in <module>
    path("", include("main.urls")),
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/urls/conf.py", line 38, in include
    urlconf_module = import_module(urlconf_module)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/src/src/dashboard/src/main/urls.py", line 61, in <module>
    urlpatterns += [path("shib/", include(shibboleth_urls, namespace="shibboleth"))]
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/urls/conf.py", line 42, in include
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Specifying a namespace in include() without providing an app_name is not supported. Set the app_name attribute in the included module, or pass a 2-tuple containing the list of patterns and app_name instead.
```
And by adding `SS_SHIBBOLETH_AUTHENTICATION: "true"` in the environment variables of the `archivematica-storage-service` service, recreating the container and trying to log in:

```console
[2024-02-20 07:19:42 -0800] [85] [ERROR] Error handling request /
Traceback (most recent call last):
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/middleware/locale.py", line 24, in process_request
    ) = is_language_prefix_patterns_used(urlconf)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/conf/urls/i18n.py", line 31, in is_language_prefix_patterns_used
    for url_pattern in get_resolver(urlconf).url_patterns:
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/urls/resolvers.py", line 715, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/urls/resolvers.py", line 708, in urlconf_module
    return import_module(self.urlconf_name)
  File "/pyenv/data/versions/3.9.18/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/src/storage_service/storage_service/urls.py", line 58, in <module>
    urlpatterns += [path("shib/", include(ShibbolethURLs, namespace="shibboleth"))]
  File "/pyenv/data/versions/3.9.18/lib/python3.9/site-packages/django/urls/conf.py", line 42, in include
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Specifying a namespace in include() without providing an app_name is not supported. Set the app_name attribute in the included module, or pass a 2-tuple containing the list of patterns and app_name instead.
```

**Steps to reproduce**

1. Enable Shibboleth authentication in both Archivematica and the Storage Service following https://www.archivematica.org/en/docs/archivematica-1.15/admin-manual/security/security/#shibboleth-setup
2. Try to authenticate on both services and you'll get `Internal Server` errors with logs similar to the above.

**Your environment (version of Archivematica, operating system, other relevant details)**

https://github.com/artefactual/archivematica/commit/d59a75fee9b3a0def6a83cb2fc9fa84330c934ca

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] A testing plan for this issue has been implemented and passed (testing plan information should be included in the issue body or comments)
- [ ] Documentation regarding this issue has been written and merged (if applicable)
- [ ] Details about this issue have been added to the release notes (if applicable)

**Comment by replaceafill** [comment]
Logging in and other Django commands in the Dashboard work now with Shibboleth authentication enabled.

**Comment by misilot** [comment]
Just wanted to check if this was part of 1.16? I didn't see mention of in the release notes so wasn't sure. Thanks!

**Comment by replaceafill** [comment]
@misilot Yes, the fix has been included in the 1.16 release.
